### PR TITLE
remove imx500 tutorials

### PR DESCRIPTION
**Pull request by Idan-BenAmi**
## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/README.md [hunk 1]
```diff
@@ -9,18 +9,121 @@ Learn how to quickly quantize pre-trained models using MCT's post-training quant
...
-We will cover various tasks and demonstrate the necessary steps to achieve efficient quantization for optimal
-deployment performance.
-- [MCT IMX500 notebooks](notebooks/imx500_notebooks/README.md)
+<details id="keras-ptq">
+  <summary>Post-Training Quantization (PTQ)</summary>
+  
+  | Tutorial                                                                                   | Included Features                                                                                   |
+  |--------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
+  | [Basic Post-Training Quantization (PTQ)](keras/example_keras_post-training_quantization.ipynb)          | &#x2705; PTQ                                                                                        |
+  | [MobileNetV2](../imx500_notebooks/keras/example_keras_mobilenetv2_for_imx500.ipynb)        | &#x2705; PTQ                                                                                        |
```
Comment (ofirgo) [inline tutorials/README.md:27]: there are several tutorials here that refer to IMX500 tutorials, they need to be removed

File: tutorials/resources/utils/keras_tutorial_tools.py [hunk 2]
```diff
@@ -1,164 +0,0 @@
-# Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
```
Comment (ofirgo) [inline tutorials/resources/utils/keras_tutorial_tools.py:1]: are you sure that none of the feature tutorials use the utils? (keras or pytorch)

**Review by ofirgo — CHANGES_REQUESTED** [review]
